### PR TITLE
finished cleaning dependencies on collection side

### PR DESCRIPTION
**Pull request by chiahaoliu**
make logic of differentiating experiment/simulation happen during startup process

### CODE REVIEW

**Comment by sbillinge** [comment]
@chiahaoliu I'm having trouble resolving conflicts here and getting tests to pass.  I think I can fix it but not right now.  Please don't work on this and I will push when I have it working.

**Comment by chiahaoliu** [comment]
I also found I didn't successfully add and push commits I did earlier today. Last commit is to share the last few changes. I will wait for your push :)

**Comment by chiahaoliu** [comment]
Updated logic based on discussion through phone this afternoon has been pushed. Code passes all unittests and running simulation(run `ipython --profie=collectiopn` locally with new `999-load.py`) is also possible.

File: xpdacq/glbl.py [hunk 1]
```diff
@@ -79,39 +61,92 @@ class glbl():
...
-    xpdRE.md['owner'] = OWNER
-    xpdRE.md['beamline_id'] = BEAMLINE_ID
-    xpdRE.md['group'] = GROUP
-    
-    if hostname != BEAMLINE_HOST_NAME:
-        shutter = mock_shutter()
+    '''
+    # logic to assign correct objects depends on simulation or real experiment
+    hostname = socket.gethostname()
+    if hostname == BEAMLINE_HOST_NAME:
```
Comment (tacaswell) [inline xpdacq/glbl.py]: This is a _super_ sketchy/brittle way to do this.  A better way might to look for an env like `'XPD_REAL_COLLECTION_ENV'` environmental variable that you can make sure you set as part of a bash script you use to launch ipython

Comment (sbillinge) [inline xpdacq/glbl.py]: @tacaswell Yes, I agree.  For now this works, but it would be good to do it with an environment variable.

Comment (chiahaoliu) [inline xpdacq/glbl.py]: We need support for advanced stuffs . When could the next meeting :p ?

**Pull request merged**